ostd/src/mm/heap_allocator/mod.rs

```diff
@@ -118,11 +118,11 @@ fn rescue(heap: &mut SpinLockGuard<Heap, LocalIrqDisabled>, layout: &Layout) ->
     // SAFETY: the frame is allocated from FrameAllocator and never be deallocated,
     // so the addr is always valid.
     unsafe {
-        debug!(
-            "add frames to heap: addr = 0x{:x}, size = 0x{:x}",
-            vaddr,
-            PAGE_SIZE * num_frames
-        );
+        // debug!(
+        //     "add frames to heap: addr = 0x{:x}, size = 0x{:x}",
+        //     vaddr,
+        //     PAGE_SIZE * num_frames
+        // );
```

Comment:
Uncomment these lines or remove these lines.

Reply:
Oh, these changed lines seems to be mis-committed, because the heap allocator be in deadlock when rescue with these `debug!` statements enabled. I'll revert them since they are not strongly related to RISC-V.

Reply:
I think that it has been fixed by #1368